lib/src/components/NxNexusPageHeader/NxNexusPageHeader.scss

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-page-header--nexus {
+  .nx-page-header__inner {
+    padding-left: 0;
+  }
+
+  .nx-product {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: 72px 16px 1fr;
+    grid-template-columns: 72px 1fr;
+    -ms-grid-rows: 39px 0 auto;
+    grid-template-rows: 39px auto;
+    gap: 0 16px;
+    grid-template-areas:
+      "logo top-row"
+      "logo bottom-row";
+
+    height: 72px;
+    margin-bottom: 0;
+  }
+
+  .nx-product__top-row {
+    -ms-grid-row-align: start;
+    align-self: start;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 1;
+    -ms-grid-column: 3;
+    grid-area: top-row;
+    padding-top: $nx-spacing-xs;
+  }
+
+  .nx-product__bottom-row {
+    -ms-grid-row: 2;
+    -ms-grid-row-span: 1;
+    -ms-grid-column: 3;
+    -ms-grid-row-align: start;
+    align-self: start;
+    grid-area: bottom-row;
+    padding-bottom: $nx-spacing-xs;
+  }
+
+  .nx-product__logo {
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 2;
+    -ms-grid-column: 1;
+    grid-area: logo;
+  }
+
+  .nx-product__link {
+    display: block;
+    height: 72px;
```

Comment:
Not necessary if you put `display: block` on the `.nx-product__logo-image`

Reply:
> Not necessary if you put `display: block` on the `.nx-product__logo-image`

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/eadfa49ad874a3cebd887045b5bce1914b7674a8

Reply:
> > Not necessary if you put display: block on the .nx-product__logo-image
>
> eadfa49

Now the `display: block` on `.nx-product__link` isn't necessary either.

Reply:
> > > Not necessary if you put display: block on the .nx-product__logo-image
> > 
> > 
> > [eadfa49](https://github.com/sonatype/sonatype-react-shared-components/commit/eadfa49ad874a3cebd887045b5bce1914b7674a8)
> 
> Now the `display: block` on `.nx-product__link` isn't necessary either.

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/47865f8d0bdb800e55dcbb45d37d56dce4302ae3